packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -374,10 +385,15 @@ export default function VisualEditor( { styles } ) {
 						{ ! isTemplateMode && (
 							<div
 								className={ classnames(
+									// This wrapper div should have the same
+									// classes as the block list beneath.
+									'is-root-container',
+									'block-editor-block-list__layout',
 									'edit-post-visual-editor__post-title-wrapper',
 									{
 										'is-focus-mode': isFocusMode,
-									}
+									},
+									blockListLayoutClass
```

Comment:
For TT3, these classes are enough to restore the padding, but some themes use `block-editor-block-list__layout` or `is-root-container`. We should just make sure we use the same classes as the root block list.

Reply:
Ideally, we also render the post title as a proper block, but that's a bit of a challenge right now.